lib/smart_answer_flows/overseas-passports/outcomes/getting_your_passport/_collect_in_person.govspeak.erb

```diff
@@ -0,0 +1,2 @@
+
+You must collect in person.  If you’re unable to, someone else can go on your behalf.
```

Comment:
Minor: The language here seems a bit odd to me. If "You *must* collect in person", then it seems odd that "someone else can go on your behalf". It might be worth double-checking with the Content Team.